### PR TITLE
Add porcelain around generating a new config for a Hush node

### PR DESCRIPTION
**Pull request by leto**
This creates a `bin/genhushconfig` bash script that is idempotent, i.e. it can always be run and it will either create ~/.hush for the first time or do nothing.

This looks and feels a lot better in our INSTALL.md as well

### CODE REVIEW

**Comment by leto** [comment]
My repo has been doing really odd stuff with that `contrib/debian/changelog` file, I will try to regen this without that....

**Comment by madbuda** [comment]
Shouldn't we put a check in so we do not overwrite an existing hush.conf?
Today not that big of a deal without wallet encryption, but I can see this being an issues when we overwrite their username/password

**Comment by leto** [comment]
@madbuda sounds good, i can move into zcutil and check for an existing hush.conf. I also need to check for symlinks, right now only a real ~/.hush dir will be found

**Comment by leto** [comment]
@madbuda would it be more useful to `exit 1` if `~/.hush` exists, so we can more easily detect that case?

**Comment by madbuda** [comment]
@leto maybe give them an options if we find an existing config to overwrite it with a warning that it will change user and password.

We should also add a few nodes for them

```
addnode=us.madmining.club
addnode=183.89.226.141
addnode=163.172.70.26
addnode=76.188.211.191
addnode=173.212.196.148
addnode=78.94.32.197
addnode=77.78.12.89
addnode=78.46.88.113
addnode=163.172.70.26
addnode=192.99.3.29
addnode=144.76.62.229
addnode=163.172.64.208
addnode=37.187.167.145
addnode=[2607:5300:60:341d::1]
addnode=[2001:41d0:d:691::]
addnode=[2a02:c207:2008:8708::1]
addnode=[2001:470:1f09:3b6:7285:c2ff:fe0e:1664]
addnode=[2a02:908:162:67dc:225:90ff:fe50:2b9b]
```

**Comment by madbuda** [comment]
@leto thinking some more, why not just make an install script?? instead of run fetch-params.sh, run make-config.sh, run build.sh

we just say clone and run install-hush-$DISTRO.sh

**Comment by leto** [comment]
@madbuda I was building up to that. I would prefer to have another install wrapper that calls this script, and keep this script small + simple, since it is pretty useful by itself.

File: INSTALL.md [hunk 1]
```diff
@@ -31,12 +31,15 @@ sudo apt-get install \
 ```
 
 Create a HUSH configuration file (*important*):
+
 ```
-mkdir -p ~/.hush
-echo "rpcuser=username" >> ~/.hush/hush.conf
-echo "rpcpassword=`head -c 32 /dev/urandom | base64`" >>~/.hush/hush.conf
+./bin/genhushconfig
```
Comment (madbuda) [inline INSTALL.md]: ./zcutil/genhushconfig

File: zcutil/genhushconfig [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Generate a new Hush node config
+
+DIR="$HOME/.hush"
+if [ -d "$DIR" ]; then
```
Comment (madbuda) [inline zcutil/genhushconfig]: should we check for hush.conf

Comment (leto) [inline zcutil/genhushconfig]: @madbuda I was thinking about that. What edge case are we trying to detect? An existing but empty .hush/ ? Or a .hush/ dir with NO hush.conf (slightly different). If you tell me the edge cases to detect and what should be done, I can try my best. I am not sure what to do if .hush/ exists but hush.conf does not. You need to check if every single possible file/dir exists, to detect a partial/corrupt copied .hush/ directory. Maybe just check for hush.conf and wallet.dat ?

**Comment by leto** [comment]
FYI this code needs to handle `.hush/` being a symlink before being merged

**Comment by madbuda** [comment]
Just checking to see if there is a plan to finish this or should we just close it?

**Comment by leto** [comment]
Not moving forward with this

**Pull request closed without merging**